automerge-protocol/Cargo.toml

```diff
@@ -14,6 +14,7 @@ uuid = { version = "^0.8.2", features=["v4"] }
 thiserror = "1.0.16"
 serde = { version = "^1.0", features=["derive"] }
 arbitrary = { version = "1", features = ["derive"], optional = true }
+tinyvec = { version = "1.2.0", features = ["alloc"] }
```

Comment:
Have you checked that this builds okay in a `no_std` environment? It looks like it should but I'm not certain.

Reply:
I don't think automerge builds in a `no_std` environment for a while (at least running `cargo nono check --package automerge` gives lots of issues). Is there a concrete use case for use in `no_std` at the moment? I can tell it might be nice but at this point I think it will be quite a lot of work to get there.

Reply:
Fair. Let's not worry about that now then.